js/templates/mobile-nav.hbs

```diff
@@ -78,31 +78,20 @@
 
 <div id="nav-advanced-receipts" class="site-nav__panel site-nav__panel--sub" aria-hidden="true">
   <button class="site-nav__back button--back button--alt-primary js-panel-close" aria-controls="nav-advanced-receipts">Advanced data</button>
-  <h3 class="site-nav__title"><a class="site-nav__link--underline" href="{{webAppUrl}}/receipts/?is_individual=true&max_date={{today}}">Receipts - all</a></h3>
+  <h3 class="site-nav__title"><a class="site-nav__link--underline" href="{{webAppUrl}}/receipts/?max_date={{today}}">Receipts - all</a></h3>
   <ul>
-    <li class="site-nav__item site-nav__item--sub"><a class="site-nav__link" href="{{webAppUrl}}/receipts/individual-contributions/?is_individual=true&max_date={{today}}">Individual contributions</a></li>
+    <li class="site-nav__item site-nav__item--sub"><a class="site-nav__link" href="{{webAppUrl}}/receipts/individual-contributions/?max_date={{today}}">Individual contributions</a></li>
     <li class="site-nav__item site-nav__item--sub is-disabled"><a class="site-nav__link">Contributions from other committees</a></li>
   </ul>
 </div>
 
 <div id="nav-advanced-spending" class="site-nav__panel site-nav__panel--sub" aria-hidden="true">
   <button class="site-nav__back button--back button--alt-primary js-panel-close" aria-controls="nav-advanced-spending">Advanced data</button>
-  <h3 class="site-nav__title"><a class="site-nav__link--underline" href="{{webAppUrl}}/receipts/?is_individual=true&max_date={{today}}">Spending</a></h3>
+  <h3 class="site-nav__title"><a class="site-nav__link--underline" href="{{webAppUrl}}/receipts/?max_date={{today}}">Spending</a></h3>
   <ul>
     <li class="site-nav__item site-nav__item--sub"><a class="site-nav__link" href="{{webAppUrl}}/disbursements/?max_date={{today}}">All disbursements</a></li>
     <li class="site-nav__item site-nav__item--sub is-disabled"><a class="site-nav__link">Operating expenditures</a></li>
     <li class="site-nav__item site-nav__item--sub is-disabled"><a class="site-nav__link">Expenditures to committees</a></li>
     <li class="site-nav__item site-nav__item--sub"><a class="site-nav__link" href="{{webAppUrl}}/independent-expenditures?max_date={{today}}">Independent expenditures</a></li>
   </ul>
 </div>
-
-<div id="nav-advanced-filings" class="site-nav__panel site-nav__panel--sub" aria-hidden="true">
-  <button class="site-nav__back button--back button--alt-primary js-panel-close" aria-controls="nav-advanced-filings">Advanced data</button>
-  <h3 class="site-nav__title"><a class="site-nav__link--underline" href="{{webAppUrl}}/receipts/?is_individual=true&max_date={{today}}">Reports and filings</a></h3>
```

Comment:
Sorry, this shouldn't be removed. The URL should just be fixed to go to `/filings/` instead of `/receipts/`